### DISCUSSION TITLE
Webapp manager

### DISCUSSION THREAD

**Discussion by edmundlaugasson** [elementary/triage#398] [Ideas]
Would propose to use [webapp-manager](https://github.com/linuxmint/webapp-manager) found in Linux Mint, which is also based on Ubuntu LTS. Very handy to create desktop app icon with custom URL, icon, name, description. Today we have more and more apps moving to web and quite often we need such a webapp per URL.

**Comment by lenemter** [comment]
This feature is already build-in into elementary OS web browser.

**Comment by teamcons** [comment]
Considering how theres no desktop app for several social media, and how common it is to be active on one, id rather suggest the welcome/first launch app suggest, with checboxes, to install the most popular ones as webapp

I dont know how difficult that would be technically ? It would be mostly copying desktop app into a config folder, and that would be a nice touch.
